packages/api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actual-app/api",
-  "version": "6.0.1",
+  "version": "7.0.0",
```

Comment:
Why the major release?

Reply:
Category hiding introduced a DB migration. We need a breaking change release for DB migrations, right? That's from what I last remember, but maybe I'm wrong.